dotCMS/src/main/java/com/dotmarketing/util/XMLUtils.java

```diff
@@ -17,4 +28,76 @@ public static String xmlEscape(String str) {
 	        return org.apache.commons.lang.StringEscapeUtils.escapeXml(str);
 	   
 	}
+	
+    public static final String prologue = "<?xml version=\"1.1\" encoding=\"UTF-8\"?>\n";
+	
+	public static boolean needsPrologue(File file) {

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'file' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5632525)